personio-personnel-data-api-oa3.yaml

```diff
@@ -1656,15 +1656,15 @@ paths:
         - name: limit
           in: query
           required: false
-          description: Pagination attribute to limit how many attendances is returning per page
+          description: Pagination attribute to limit the number of absence periods per page
           schema:
             type: integer
             minimum: 1
             default: 200
         - name: offset
           in: query
           required: false
-          description: Pagination attribute to identify which page you are requesting, by the form of telling an offset from the first record that would be returned.
+          description: Pagination attribute to identify which page number you are requesting
```

Comment:
should we mention - `, starting from zero` or `, starting from 1`? because usually the pages go `1..n` and the schema here (lines below) says `0..n`

Reply:
and the description says "page number"

Reply:
0 -> page 1 in the current implementation, so I don't think it's necessary to add that